<launch>

    <arg name="offset_topic" default="/layer_center/offset_mm" />
    <arg name="path_topic" default="/ur_path_original" />
    <arg name="index_topic" default="/path_index" />
    <arg name="cmd_topic" default="/mur620a/UR10_r/twist_fb_command" />

    <!-- Node that runs the path transformer script -->
    <node name="profile_center_offset_controller" pkg="laser_scanner_tools" type="profile_center_offset_controller.py" output="screen">
        <!-- Transformation parameters -->
        <param name="offset_topic" value="$(arg offset_topic)" />
        <param name="path_topic" value="$(arg path_topic)" />
        <param name="index_topic" value="$(arg index_topic)" />
        <param name="cmd_topic" value="$(arg cmd_topic)" />
    </node>
</launch>

